c_horiz_interp/include/c_horiz_interp_get_weights.inc

```diff
@@ -0,0 +1,97 @@
+function CFMS_HORIZ_INTERP_GET_WEIGHTS_2D_(lon_in_ptr, lat_in_ptr, lonlat_in_shape,&
+     lon_out_ptr, lat_out_ptr, lonlat_out_shape, mask_in_ptr, mask_out_ptr, &
+     interp_method, verbose, max_dist, src_modulo, is_latlon_in, is_latlon_out) &
+     bind(C, name=CFMS_HORIZ_INTERP_GET_WEIGHTS_2D_BINDC_)
+
+  implicit none
+
+  type(c_ptr), intent(in), value :: lon_in_ptr
+  type(c_ptr), intent(in), value :: lat_in_ptr
+  integer, intent(in) :: lonlat_in_shape(2)
+  type(c_ptr), intent(in), value :: lon_out_ptr
+  type(c_ptr), intent(in), value :: lat_out_ptr
+  integer, intent(in) :: lonlat_out_shape(2)
+  type(c_ptr), intent(in), value :: mask_in_ptr
```

Comment:
The ordering of arguments were changed.  The optional arguments come after the required arguments.